### PR TITLE
HBW-297 Make font-size relative to the base

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: hbw/app/assets/stylesheets/hbw/variables.sass [hunk 1]
```diff
@@ -26,3 +26,5 @@ $btn-danger-bg: transparent
 $btn-danger-color: $btn-danger-border
 
 $deadline-red: $brand-danger
+
+$font-size-normal: 15px
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/variables.sass:30]: Let's convert this to CSS-variable like `--base-font-size` and use it in branding styles right now :)

**Comment by mxsmirnoff** [comment]
HOPER seems to be ok.

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass [hunk 2]
```diff
@@ -18,29 +18,41 @@
...
+  --base-hbw-text-color: #{$text-color}
   --base-hbw-priority-medium-color: #926902
   --base-hbw-priority-high-color: #{$brand-warning}
   --base-hbw-urgent-color: #E74C3C
   --base-hbw-task-overview-first-child-color: #4A4A4A
   --base-hbw-short-list-row-hover: #{$table-bg-hover}
   --base-hbw-short-list-expired-color: #BD2025
   --base-hbw-li-danger: #E74C3C
+
+  --base-font-size: #{$font-size}
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/brand_variables.sass]: This should be in the `common` group.

File: hbw/app/assets/stylesheets/hbw.sass [hunk 3]
```diff
@@ -17,6 +17,9 @@
 
   @import 'hbw/branding'
 
+  body
+    font-size: $font-size
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: I think we should do it only once: https://github.com/latera/homs/pull/480/files#diff-44af70099daac6ae6847f1ac32d66ab6R4

File: hbw/app/assets/stylesheets/hbw/variables.sass [hunk 4]
```diff
@@ -26,3 +26,5 @@ $btn-danger-bg: transparent
 $btn-danger-color: $btn-danger-border
 
 $deadline-red: $brand-danger
+
+$font-size: 15px
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/variables.sass]: Use CSS-variables only if it's possible.

**Pull request merged**